python/pyspark/sql/utils.py

```diff
@@ -193,6 +193,15 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
     return cast(FuncT, wrapped)
 
 
+def require_spark_context_initialized() -> SparkContext:
+    """Raise RuntimeError if SparkContext is not initialized,
+    otherwise, returns the active SparkContext."""
+    sc = SparkContext._active_spark_context
+    if sc is None or sc._jvm is None:
+        raise RuntimeError("SparkContext must be initialized for JVM access.")
```

Comment:
I would just say something like: `SparkContext or SparkSession should be created first` instead

Reply:
Updated.